src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,257 @@
 ---
-title: Reduce data ingest
+title: Reduce data ingested
 tags:
-  - Integrations
-  - Kubernetes integration
-  - Understand and use data
-metaDescription: 'New Relic''s Kubernetes integration: How to install and activate the integration, and what data is reported.'
+    - Integrations
+    - Kubernetes integration
+    - Understand and use data
+metaDescription: 'New Relic''s Kubernetes integration: How to reduce the data ingested.'
 redirects:
-  - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
-  - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
-  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
-freshnessValidatedDate: never
+    - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
+    - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+    - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
+freshnessValidatedDate: 2024-06-24
 ---
 
-## Reduce data ingest [#reducedataingest]
+If you've checked your data ingestion and you'd like to reduce the amount you have, you can do it through charts and through parameters.
```

Comment:
I changed a little bit the content, because those settings are related only to Helm Chart. If you want to change directly in a Manifest YAML file is more complex, for example low Data Mode parameter in the Helm Chart does several changes in the final Manifest YAML file applied.

Here is my proposal:


If you've reviewed your data ingestion (https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-coming-new-relic/) and want to reduce the volume of data you're sending, we've got a simple solution for you. You can easily achieve this by setting up our nri-bundle (https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) Helm chart. Our charts provide a way to fine-tune your settings and reduce the amount of data ingested, which may result in slightly less detailed information. These setting, also allow you to focus on the most critical data points, helping you identify the most important insights

Reply:
Done!